build.gradle

```diff
@@ -94,7 +94,7 @@ dependencies {
 
     implementation 'com.exactpro.th2:common:5.4.0-dev'
     implementation 'com.exactpro.th2:common-utils:2.2.0-dev'
-    implementation 'com.exactpro.th2:grpc-act-template:4.1.0-TH2-5073+'
+    implementation 'com.exactpro.th2:grpc-act-template:4.2.0-dev'
```

Comment:
Actually, this was also published as `4.2.0`. So, probably we should use release version instead of dev-release